R/opts.R

```diff
@@ -638,9 +638,23 @@ obs_opts <- function(family = c("negbin", "poisson"),
       .frequency = "regularly",
       .frequency_id = "obs_opts"
     )
-  #nolint end
+  } else {
+    cli_inform(
+      c(
+        "i" = "{col_red(\"Explicit NA values will be treated as missing as
+        compared to being treated as zero observations in previous
+        versions.\")}",
+        "i" = "If you prefer for NA's to be treated as zero observations, see
+        solutions in {.url https://github.com/epiforecasts/EpiNow2/issues/767#issuecomment-2348805272}", #nolint
```

Comment:
We need a vignette on modelling data reported on a less regular interval than daily. We can link that here instead of this comment.

Reply:
This has been logged in #772.